aws/internal/keyvaluetags/generators/updatetags/main.go

```diff
@@ -110,6 +111,7 @@ func main() {
 	}
 	templateFuncMap := template.FuncMap{
 		"ClientType":                      keyvaluetags.ServiceClientType,
+		"TagFunctionClass":                ServiceTagFunctionClass,
```

Comment:
Similar suggestion as above:

```suggestion
		"TagPackage":                      ServiceTagPackage,
```